src/Apps/Purchase/Components/PurchaseHistory.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Image,
+  Link,
+  Sans,
+  Serif,
+  StackableBorderBox,
+} from "@artsy/palette"
+import { PurchaseHistory_orders } from "__generated__/PurchaseHistory_orders.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface OrderRowProps {
+  order: any
```

Comment:
We should properly type this! Something like `PurchaseHistory_orders["edges"][number]["node"] I _think_ can work.